scripts/skills/perks/perk_rf_discovered_talent.nut

```diff
@@ -1,6 +1,6 @@
 this.perk_rf_discovered_talent <- ::inherit("scripts/skills/skill", {
 	m = {
-		IsApplied = false
+		AttributesRolled = []
```

Comment:
Is the order in this array relevant?
You could prefill it with `false` and have instead the index in the array represent an attribute from `::Const.Attributes`. That should simplify the code overall.